content/recipes/applied-examples/eubopen.md

```diff
@@ -0,0 +1,51 @@
+(fcb-eubopen)=
+# FAIRification of High-Content Screening data using Bioimage Archive
+
+````{panels_fairplus}
+:identifier_text: TODO
+:identifier_link: TODO
+:difficulty_level: 3
+:recipe_type: applied_example
+:reading_time_minutes: 15
+:intended_audience: data_manager  
+:has_executable_code: nope
+:recipe_name: FAIR High-Content Screening data
+```` 
+
+
+## Abstract
+
+You run high-content screens to test the effect of compounds on cells. You want to make your data FAIR. This recipe is for you. 
+
+
+## Recipe
+
+1. Download the Python scripts and the metadata template Excel sheet from [https://doi.org/10.5281/zenodo.6325622](https://doi.org/10.5281/zenodo.6325622).
+2. Organize your metadata according to the metadata template sheet.
```

Comment:
i would state what the template is based on if necessary, or add a sentence in brackets or such)

Reply:
describe how this template was created ? was it supplied to you by Bioimage? was it custom made ? Is the template identified, versioned etc..?